openshift/templates/deploy.dc.yaml

```diff
@@ -136,7 +122,7 @@ objects:
         app: ${NAME}-${SUFFIX}
       name: ${NAME}-${SUFFIX}-source
     spec:
-      host: ${NAME}-${SUFFIX}-source-${PROJECT_DEPLOY}.pathfinder.gov.bc.ca
+      host: ${NAME}-${SUFFIX}.pathfinder.gov.bc.ca
       port:
         targetPort: 8080-tcp
```

Comment:
@DerekRoberts we need to use different port other than 8080 for secure routes. We could not send POST request when https route was pointing to 8080-tcp.

Reply:
@Nkuruba My work is the part in red and green.  Still, that's good info and will go into a ticket right after this!